frontend/src/pages/learningCenter/LearningCenterToolbar.tsx

```diff
@@ -216,32 +222,46 @@ const LearningCenterToolbar: React.FC<LearningCenterToolbarProps> = ({
           <>
             <ToolbarItem data-testid="resources-select-type">
               <Select
-                variant={SelectVariant.single}
                 aria-label="Select sort type"
+                onOpenChange={(isOpen) => setIsSortTypeDropdownOpen(isOpen)}
                 isOpen={isSortTypeDropdownOpen}
-                onToggle={(e, isEnabled) => setIsSortTypeDropdownOpen(isEnabled)}
-                placeholderText={`Sort by ${sortTypes[sortType]}`}
+                toggle={(toggleRef) => (
+                  <MenuToggle
+                    ref={toggleRef}
+                    aria-label="Options menu"
```

Comment:
That should only be true if there's no text within the button. I can see there being a violation for the sort order but not the sort type.
Can you share your error? I did a scan in browser with axe devtools and didn't see any error if it was removed from sort type.

Reply:
@christianvogt Cypress a11y integration is catching this
I removed the aria-label for sortType and this is what I got 
<img width="1482" alt="Screenshot 2024-07-04 at 10 09 24 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/99238909/ca8e5ded-54e1-48ef-a625-f27ff06ef8c1">

Reply:
This error makes sense because you removed the aria-label on the order type. Order menu toggle is just an icon, it has no text. Therefore it requires an aria-label.

Since all the old menu toggles used `options menu` as the default aria-label, we can continue to keep it, but i just was noting that it's odd and so worth fixing or creating a separate issue to improve later.